src/rail/estimation/algos/deepdisc.py

```diff
@@ -295,5 +422,5 @@ def finalize(self):
         qp_distn = self.calculate_point_estimates(qp_distn)
         self.add_handle("output", data=qp_distn)
         truth_dict = dict(redshift=self.true_zs)
-        #truth = DS.add_data("truth", truth_dict, TableHandle)
+        # truth = DS.add_data("truth", truth_dict, TableHandle)
```

Comment:
Also should remove this comment unless we're really going to want to uncomment it in the future.